react-list.es6

```diff
@@ -383,7 +393,7 @@ module.exports = class ReactList extends Component {
     }
   }
 
-  getSizeOf(index) {
+  getSizeOfRow(index) {
```

Comment:
I actually purposefully avoided this wording because for lists on the `x` axis this method means "get size of column". Let's just leave this naming as is for now.